doc/contributing/CODING_STANDARDS.md

```diff
@@ -117,7 +120,9 @@ OPENASSETIO_DECLARE_PTR(Host)
 
 class OPENASSETIO_CORE_EXPORT Host final {
 public:
-    static HostPtr make(...);
+    OPENASSETIO_ALIAS_PTR(Host)
+
+    static Ptr make(...);
```

Comment:
[Consider][Minor] Do you think maybe leaving the more descriptive type when working concretely (ie HostPtr instead of Ptr) might be clearer, means the reader dosen't have to burrow into a macro. (or rather, only has to burrow into one).

Reply:
Good point, I had the same thought with the rest of the codebase, but obviously didn't change the .md to match.

Reply:
Updated docs in c0566f8b